assets/dev/scss/editor/preview/preview.scss

```diff
@@ -120,6 +120,7 @@ html.elementor-html {
 	> .elementor-element-overlay {
 		position: absolute;
 		top: 0;
+		width: 100%;
```

Comment:
Explain

Reply:
We actually don't need that anymore, that was a leftover from some experiment.